src/parser/browser/rich-text-browser-parser.ts

```diff
@@ -1,45 +1,44 @@
 import {
-convertDomNodeAttributes,
-getAllNewLineAndWhiteSpace,
-isElementNode,
-isRootNode,
-isTextNode
-} from "../../utils/index.js"
+  convertDomNodeAttributes,
+  getAllNewLineAndWhiteSpace,
+  isElementNode,
+  isRootNode,
+  isTextNode,
+  throwError,
+} from "../../utils/index.js";
 import { DomNode, ParseResult } from "../parser-models.js";
 
 export const parse = (input: string): ParseResult => {
   const parser = new DOMParser();
-  const sanitizedInput = input.replaceAll(getAllNewLineAndWhiteSpace, '');
-  
-  const document = parser.parseFromString(sanitizedInput, 'text/html');
+  const sanitizedInput = input.replaceAll(getAllNewLineAndWhiteSpace, "");
+
+  const document = parser.parseFromString(sanitizedInput, "text/html");
 
   if (isRootNode(document) && document.body.firstChild) {
     return {
       children: Array.from(document.body.children).flatMap(parseInternal),
     };
-
-  }
-  else {
-    throw new Error("Cannot parse a node that is not a root node.");
   }
-}
+
+  return throwError("Cannot parse a node that is not a root node.");
```

Comment:
sorry for confusion, I wanted you to use this function only with ternary operators :D Here I would prefer to the normal 
```ts
throw new Error("Cannot parse a node that is not a root node.")
```